packages/office-addin-mock/src/officeMockObject.ts

```diff
@@ -1,13 +1,21 @@
+import { getHostType, Host } from "./host";
 import { isValidError, PossibleErrors } from "./possibleErrors";
+import { ObjectData } from "./objectData";
 
 /**
  * Creates an office-js mockable object
  * @param object Object structure to provide initial values for the mock object (Optional)
+ * @param host Host tested by the object (Optional)
  */
 export class OfficeMockObject {
-  constructor(object?: ObjectData) {
+  constructor(object?: ObjectData, host?: Host) {
     this.properties = new Map<string, OfficeMockObject>();
     this.loaded = false;
+    if (host) {
+      this.host = host;
+    } else {
+      this.host = getHostType(object);
```

Comment:
should we be checking for a valid host?  As it stands if we get "unknown" we still continue on our merry way.

Reply:
I didn't want to make it mandatory for people to put the host option, so that's why I am not caring if someone doesn't define the host.
In this case, the default behavior is: "not outlook"

Reply:
That's an ambiguous state, but I guess it's no different than we already are.